src/components/organize/TagsManager/TagsManager.spec.tsx

```diff
@@ -125,4 +131,36 @@ describe('<TagsManager />', () => {
     // Check that callback has been called
     expect(onSelect).toHaveBeenCalledWith(tag1);
   });
+  it('can remove a tag', () => {
+    const onRemove = jest.fn((tag: ZetkinTag) => tag);
+
+    const tag1 = mockTag({
+      group: { id: 2, title: 'Skills' },
+      id: 4,
+      title: 'Phone banking',
+    });
+
+    const { getByText, container } = render(
+      <TagsManager
+        appliedTags={[tag1]}
+        availableTags={[tag1]}
+        onRemove={onRemove}
+        onSelect={selectTagCallback}
+      />
+    );
+
+    // Hover tag to remove
+    const tagOption = getByText('Phone banking');
+    hover(tagOption);
+
+    // Click delete button
+    const removeTagButton = container.querySelector(`.MuiChip-deleteIcon`);
+    expect(removeTagButton).not.toBeNull();
+    if (removeTagButton) {
```

Comment:
Is this if statement necessary? The `expect()` above should already cause the test to fail if the button could not be found in the DOM.

Reply:
This bit is just necessary because Typescript isn't smart enough to know that the expect means that it won't be null, so it's there so we can do the click action. I prefer to use conditionals to typecasting just because then types are as correct as possible. And this code still works properly.